docs/current_docs/manuals/administrator/tutorials/index.mdx

```diff
@@ -0,0 +1,12 @@
+---
+slug: /manuals/administrator/tutorials
+displayed_sidebar: "current"
+---
+
+# Tutorials
+
+Learn more about Dagger using the step-by-step walkthroughs in this section.
+
+- [Configure a Custom Registry Mirror for Dagger](./272381-custom-registry.mdx)
+- [Run Dagger on OpenShift with GitLab Runners](./488565-openshift-gitlab.mdx)
+- [Run Dagger on Amazon EKS with GitHub Actions Runner and Karpenter](./934191-eks-github-karpenter.mdx)
```

Comment:
```suggestion
```

This should be merged into 1) the Github Actions integration page, as a special optimization, and 2) a new EKS integration page.

Reply:
OK. I have removed this from the current PR and noted this in Linear for follow-up in a separate PR.